src/Tribute.js

```diff
@@ -261,6 +261,8 @@ class Tribute {
     let wrapper = this.range.getDocument().createElement("div"),
       ul = this.range.getDocument().createElement("ul");
     wrapper.className = containerClass;
+    wrapper.setAttribute('aria-expanded','true');
+    ul.setAttribute('id','tributeResults');
```

Comment:
this will cause issues when multiple instance of tribute are used in the page, because it will create duplicate ids. The uniqueness of ids must be preserved.

A solution could be to use `el.id + '-tributeResults'` (and maybe skip the id entirely if `el` does not have an id, as generating a random part for the id won't be useful to reference it from the outside). This way, the id stays unique (assuming that the page was not using duplicate ids and that it was not already using a `-tributeResults` suffix in its set of ids of course) while being deterministic and user-controlled (making it possible to reference the id in aria attributes maintained outside tribute).